test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -779,26 +786,49 @@ void testUtilV2(const TestUtilOptionsV2& options) {
   }
 }
 
-// Configure the listener with unittest{cert,key}.pem and ca_cert.pem.
-// Configure the client with expired_san_uri_{cert,key}.pem
+void updateFilterChain(
+    const envoy::extensions::transport_sockets::tls::v3::DownstreamTlsContext& tls_context,
+    envoy::config::listener::v3::FilterChain* filter_chain) {
+  filter_chain->mutable_transport_socket()->mutable_typed_config()->PackFrom(tls_context);
+}
+
+struct OptionalServerConfig {
+  absl::optional<std::string> cert_hash = {};
```

Comment:
I was following the google TOTW to use assignment-like syntax (external version https://abseil.io/tips/88).
I think there should be no copy (only one name here) and no assignment invoked(initialization not assignment). So this "copy list initialization" is same as direct list initialization (`cert_hash{}`) expect that `explicit` constructor is disallowed.

But I agree that for the empty construction case here, there is no concern about ambiguity that exists in direct list initialization. Done!

Reply:
I think Envoy style generally is:
1. Don't put any empty braces if there is no material difference.
2. Just do `a{}` rather than `a = {}` for empty at least.

Reply:
Got it! Done